### PR TITLE
retrieve default crashkernel from kdumpctl

### PR DESCRIPTION
**Pull request by coiby**
kexec-tools has removed the support for crashkernel.default and provided
"kdumpctl get-default-crashkernel [DUMP_MODE]" to retrieve default
crashkernel value.

### CODE REVIEW

**Comment by coiby** [comment]
@ryncsn Hi Kairui. If you have time, could you help review this PR?

File: com_redhat_kdump/service/installation.py [hunk 1]
```diff
@@ -48,6 +47,35 @@ def __init__(self, kernels, sysroot, kdump_enabled, fadump_enabled, reserved_mem
...
+
+        if not ck_val:
+            log.warning("Can't retrieve the default crashkernel value from "
+                        "the installed kexec-tools, try to retrieve it from "
+                        "the installer kexec-tools")
+            del args['root']
+            try:
+                ck_val = util.execWithCapture(**args)
+            except FileNotFoundError:
+                pass
```
Comment (ryncsn) [inline com_redhat_kdump/service/installation.py:70]: Maybe generate a error message here rather than fail silently.

**Comment by ryncsn** [comment]
Looks good to me except some little nit pick, nice work!

**Comment by coiby** [comment]
> Looks good to me except some little nit pick, nice work!

Thanks:)

**Pull request merged**